viewer/src/graphicalTypes.ts

```diff
@@ -0,0 +1,57 @@
+export interface Graphical {
+    ComponentViewModelGraphicalData: ComponentViewModelGraphicalData;
+}
+
+export interface ComponentViewModelGraphicalData {
+    Links: Array<Links>;
+    States: States;
+    TransitionPatternStates: Array<TransitionPatternStates>;
+    TransversalLinks: Array<TransversalLinks>;
+}
+
+export interface Links {
+    TransitionGraphicalData: Array<TransitionGraphicalDataElement>;
+}
+
+export interface TransitionGraphicalDataElement {
+    $: Properties;
```

Comment:
Why do you need to define a $ member ? Not very clear
$ is mainly by jquery stuffs..